src/App.tsx

```diff
@@ -1,24 +1,210 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames';
+import { Todo } from './types/Todo';
+import {
+  deleteTodo,
+  getTodos,
+  patchTodo,
+  postTodo,
+} from './api/todos';
+import { TodoList } from './components/TodoList';
 
-const USER_ID = 0;
+const USER_ID = 6616;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState('');
+  const [query, setQuery] = useState('');
+  const [disabledInput, setDisabledInput] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo>();
+  const [loadingIds, setLoadingIds] = useState<number[]>([]);
+
+  const closeError = () => {
+    setError('');
+  };
+
+  const addTodo = (title: string) => {
```

Comment:
let's not recreate this callback on every rerender, use some type of memoization for this and your other callbacks